Gruntfile.js

```diff
@@ -70,6 +70,12 @@ module.exports = function(grunt) {
                 return "url('" + imgUrl + "') repeat";
               }
               return 'initial';
+            },
+            assignFontSize: function(fontSize) {
+              if (fontSize) {
+                return fontSize + 'rem' || 'initial';
```

Comment:
Oops, you're right. I copied that block of text from the colors method and evidently did not edit that part.